src/PtrArrays.jl

```diff
@@ -87,4 +87,9 @@ Base.@propagate_inbounds function Base.setindex!(p::PtrArray, v, i::Int)
     p
 end
 
+# Strided array interface
+Base.strides(p::PtrArray) = Base.size_to_strides(1, size(p)...)
+Base.unsafe_convert(::Type{Ptr{T}}, A::PtrArray{T}) where {T} = p.ptr
+Base.elsize(::Type{P}) where {P<:PtrArray} = sizeof(eltype(P))
```

Comment:
```suggestion
Base.unsafe_convert(::Type{Ptr{T}}, p::PtrArray{T}) where T = p.ptr
Base.elsize(::Type{P}) where P<:PtrArray = sizeof(eltype(P))
```
Fix typo and match style elsewhere in file